proceeding/text/data.tex

```diff
@@ -31,4 +29,12 @@ \section{Data}
   \end{itemize}
 \end{enumerate}
 
-(Table of number of sources per catalog (and maybe also number of sources in each class?))
+
+\begin{figure}[h]
+  \centerline{\includegraphics[width=\textwidth]{figures/four_images}}
+  \caption{Survey images (left to right): Fermi color, Akari 90um, Planck LFI, Planck HFI.}
```

Comment:
This I find very interesting. Nice!
Maybe even show a second row of spheres?
Or have a second row with zoom-ins for the Galactic center region (or some other interesting region), like 20 x 10 deg or 10 x 5 deg, maybe even with gamma-ray sources shown?
That would already to in the direction of "science application", we would write a few sentences comparing the visible structures.